aten/src/ATen/native/cuda/BatchLinearAlgebraLib.cu

```diff
@@ -69,6 +69,34 @@ void geqrf_batched_cublas(const Tensor& input, const Tensor& tau) {
   });
 }
 
+template <typename scalar_t>
+static void apply_lu_solve_batched_cublas(const Tensor& b, const Tensor& lu, const Tensor& pivots) {
```

Comment:
`at::cuda::blas::getrsBatched` is available only for CUDA builds. This function should raise an error when cuBLAS is not available. See `apply_geqrf_batched` in the same file.

Reply:
ROCm CI build fails because of that.